commands/download_mta_op_logs_command.go

```diff
@@ -74,30 +87,49 @@ func (c *DownloadMtaOperationLogsCommand) Execute(args []string) ExecutionStatus
 		return Failure
 	}
 
+	var operationIds []string
+
+	if hasMtaId(flags) {
+		operations, err := mtaClient.GetMtaOperations(&mtaId, formatLast(last), nil)
```

Comment:
I think there is already some kind of method which is deciding how to call this:
https://github.com/cloudfoundry-incubator/multiapps-cli-plugin/blob/b775b2b2f88ee41bea738269f6675508e5404696/commands/mta_operations_command.go#L112

Reply:
Is this updated?